terms/tensorprocessingunit_pi.bib

```diff
@@ -0,0 +1,53 @@
+@inproceedings
+{
+    jouppi2017datacenter,
+    title={In-datacenter performance analysis of a tensor processing unit},
+    author={Jouppi, Norman P and Young, Cliff and Patil, Nishant and Patterson, David and Agrawal, Gaurav and Bajwa, Raminder and Bates, Sarah and Bhatia, Suresh and Boden, Nan and Borchers, Al and others},
+    booktitle={Computer Architecture (ISCA), 2017 ACM/IEEE 44th Annual International Symposium on},
+    pages={1--12},
+    year={2017},
+    organization={IEEE}
+}
+
+@inproceedings
+{
+    paun2013determinism,
+    title={On the determinism of multi-core processors},
+    author={Paun, Vladimir-Alexandru and Monsuez, Bruno and Baufreton, Philippe},
+    booktitle={French Singaporean Workshop on Formal Methods and Applications},
+    pages={35,36},
+    year={2013}
+}
+
+@misc
+{
+    sato,
+    author = {Sato, Kaz and Young, Cliff and Patterson, David},
+    title  = {An in-depth look at Google's first Tensor Processing Unit (TPU)},
+    howpublished ={Website},
+    year={2017},
+    month=5,
+    url ={https://cloud.google.com/blog/products/gcp/an-in-depth-look-at-googles-first-tensor-processing-unit-tpu}
+}
+
+@misc
+{
+    google,
+    author = {Google},
+    title  = {XLA - TensorFlow, compiled},
+    howpublished ={Website},
+    year={2017},
+    month=3,
+    url    = {https://developers.googleblog.com/2017/03/xla-tensorflow-compiled.html}
+}
+
+@misc
```

Comment:
Not a scholarly reference: find book or research article

Reply:
I'm aware that it is not, but the only scholarly references i could find specifically on TPUs is the one published. All other material is either Google's documentation on https://cloud.google.com/tpu/ or blogposts.